azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/AbstractSpringMojo.java

```diff
@@ -103,6 +105,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
 
     protected void initExecution() {
+        // Init sdk log level
+        if (getLog().isDebugEnabled()) {
+            SpringServiceUtils.setLogLevel(LogLevel.BODY_AND_HEADERS);
```

Comment:
will this introduce too many traces if users are running in debug mode -X?

Reply:
Users could see the http requests and reponses within debug mode, it could be more convenient for debugging. (And i think there are already many traces in debug mode -X, it'l okay to add a bit more...